walk-in-interview/src/main/webapp/account/firebase-auth.js

```diff
@@ -0,0 +1,240 @@
+/**
+ * This file will handle all the functions related to firebase auth.
+ *
+ * Note that the following lines would need to be added to any html
+ * file that wishes to use firebase auth:
+ * <script src="https://www.gstatic.com/firebasejs/7.17.1/firebase-app.js"></script>
+ * <script src="https://www.gstatic.com/firebasejs/7.17.1/firebase-auth.js"></script>
+ */
+
+import {AppStrings} from '../strings.en.js';
+import {setCookie, getCookie,
+  USER_TYPE_COOKIE_PARAM, USER_TYPE_NO_USER, USER_TYPE_APPLICANT} from '../common-functions.js';
+import {API} from '../apis.js';
+
+const firebaseConfig = {
+  apiKey: 'AIzaSyDhpzKNLAMNyEdw6ovQ5sPvnOhXDwhse-o',
+  authDomain: 'com-walk-in-interview.firebaseapp.com',
+  databaseURL: 'https://com-walk-in-interview.firebaseio.com',
+  projectId: 'google.com:walk-in-interview',
+  storageBucket: 'undefined',
+  messagingSenderId: '610715446188',
+  appId: '1:610715446188:web:8cec0c2798940bb9d32809',
+  measurementId: 'G-RJ55G4ZXN8',
+};
+
+firebase.initializeApp(firebaseConfig);
+
+// TODO(issue/21): get the language from the browser
+const CurrentLocale = 'en';
+
+firebase.auth().languageCode = CurrentLocale;
+
+// Firebase UI
+const ui = new firebaseui.auth.AuthUI(firebase.auth());
+
+const STRINGS = AppStrings['auth'];
+
+const Auth = {};
+
+/**
+ * This will create a new business account.
+ *
+ * @param {String} email The email for the new business account.
+ * @param {String} password The password for the new business account.
+ */
+Auth.createBusinessAccount = (email, password) => {
+  firebase.auth().createUserWithEmailAndPassword(email, password)
+      .catch((error) => {
+        console.error(error);
```

Comment:
should you alert here?

Reply:
comment is still valid.

Reply:
Will resolve in the next PR, same for below business related